tests/integration_tests/asn1_choice/resources/taste/asn1_choice.pro.user

```diff
@@ -0,0 +1,287 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Is this needed?

Reply:
This applies to all instances in tests

Reply:
It is required (it is space-creator project file)

Reply:
Ou, no, this is user file, so this isn't required.

Reply:
Done.